watertap/examples/chemistry/tests/test_phosphorus_removal.py

```diff
@@ -1055,13 +1052,10 @@ def test_scaling_simple_phosphorus_removal(self, simple_phosphorus_removal):
     def test_initialize_solver(self, simple_phosphorus_removal):
         model = simple_phosphorus_removal
 
-        # These might be unnecessary now with new changes in IDAES
-        model.fs.unit.control_volume.properties_out[0.0].log_mole_frac_phase_comp_true.setlb(-50)
-        model.fs.unit.control_volume.properties_out[0.0].log_mole_frac_phase_comp_true.setub(0.001)
-        model.fs.unit.control_volume.properties_out[0.0].mole_frac_phase_comp.setub(1.001)
+        # if the initialization is sufficiently good, it may be
+        # a better idea to do this manually before the solve
+        iscale.constraint_autoscale_large_jac(model.fs.unit)
```

Comment:
Was this step necessary for the phosphorus removal test case? Would it not solve without it?

Reply:
Yes -- phosphorus removal is the one case that didn't "just work". You might also notice the slightly larger variable scaling. The Jacobian scaling at the initial values is much better than the resulting Jacobian scaling after the control volume initialization but before the whole unit initialization.

That said, I did not explore deeply other options, e.g., setting `bound_push` lower. It's also possible the approach for the chemistry models needs to be somewhat adjusted if we adopt this solver.

Reply:
In all likely hood, if this is the only chemistry test giving issues, then I think we are fine with the new solver. 

This test can actually be removed (it is on my to do list), as it is now an obsolete representation of the phosphorus removal process, which is being tested in 'test_pH_dependent_solubility.py'

Reply:
@bknueven For your reference, #258 is my issue containing the list of chemistry tests that are outdated and will be removed at some point.